src/odemis/gui/cont/correlation.py

```diff
@@ -0,0 +1,280 @@
+
+    def add_streams(self, streams: list):
+        """add streams to the correlation tab
+        param streams: (list[StaticStream]) the streams to add"""
+
+        # NOTE: we only add streams if they are not already in the correlation tab
+        # we will not remove streams from the correlation tab from outside it, 
+        # as the user may still want to correlate them, 
+        # even if they have been removed from another tab, e.g. 'localization'
+
+        # add streams to correlation tab
+        logging.debug(f"Adding {len(streams)} streams to correlation tab")
+        for s in streams:
+            
+            # skip existing streams
+            if s in self._tab_data_model.streams.value:
+                continue 
+
+            # reset the stream correlation data, and add to correlation streams
+            self._reset_stream_correlation_data(s)
+            self._tab_data_model.streams.value.append(s)          
+            
+            # add stream to streambar
+            sc = self._tab.streambar_controller.addStream(s, add_to_view=True, play=False)
+            sc.stream_panel.show_remove_btn(True)  
+
+    def clear_streams(self):
+        """clears streams from the correlation tab"""
+        self._tab.streambar_controller.clear()
+
+    def _on_mouse_down(self, evt):
+        """handle mouse down events"""
+        
+        active_canvas = evt.GetEventObject()
+        logging.debug(f"mouse down event, canvas: {active_canvas}")
+ 
+        # check if shift is pressed, and if a stream is selected
+        if evt.ShiftDown() and self.selectedStream.value is not None:
+                                    
+            # get the position of the mouse, convert to physical position
+            pos = evt.GetPosition()
+            p_pos = active_canvas.view_to_phys(pos, active_canvas.get_half_buffer_size())
+            logging.debug(f"shift pressed, mouse_pos: {pos}, phys_pos: {p_pos}")
+
+            # move selected stream to position
+            self._move_stream_to_pos(p_pos)
+    
+        else:
+            active_canvas.on_left_down(evt)       # super event passthrough      
+
+    def _on_char(self, evt):
+        """handle key presses"""
+        _key = evt.GetKeyCode()
+        _shift_mod = evt.ShiftDown()
+
+        # correlation control modifiers
+        dx = self._panel.dx_step_cntrl.GetValue() * 1e-6
+        dy = self._panel.dy_step_cntrl.GetValue() * 1e-6
+        dr = numpy.deg2rad(self._panel.dr_step_cntrl.GetValue()) 
+        dpx = self._panel.dpx_step_cntrl.GetValue() / 100
+
+        ### CONTROLS ##############################
+        # SHIFT + LEFT CLICK -> MOVE_TO_POSITION
+        # LEFT, RIGHT -> TRANSLATION X
+        # UP, DOWN -> TRANSLATION Y
+        # SHIFT + LEFT, RIGHT -> ROTATION
+        # SHIFT + UP, DOWN -> SCALE 
+        ###########################################
+
+        if _shift_mod:
+            dx, dy = 0, 0
+        else:
+            dr, dpx = 0, 0
```

Comment:
I feel it's an inverted logic. Can you start by default all the values are set to 0, and based on shift, read just the right values from the widgets?